items/COUNTER_STRIKE;3.json

```diff
@@ -0,0 +1,28 @@
+{
+  "itemid": "minecraft:enchanted_book",
+  "displayname": "§aEnchanted Book",
+  "nbttag": "{ench:[],overrideMeta:1b,HideFlags:254,display:{Lore:[0:\"§9Counter-Strike III\",1:\"§7Gain §a+6❈ Defense §7for §a7s\",2:\"§a§7on the first hit from an\",3:\"§7enemy.\",4:\"\",5:\"§7Use this on an item in an Anvil\",6:\"§7to apply it!\",7:\"\",8:\"§6Source:\",9:\"§aV: §7Dark Auction\",10:\"\",11:\"§6Applied To:\",12:\"§7- §fChestplate\",13:\"\",14:\"§a§lUNCOMMON\"],Name:\"§aEnchanted Book\"},ExtraAttributes:{id:\"COUNTER_STRIKE;5\",enchantments:{counter_strike:5}}}",
+  "damage": 0,
+  "lore": [
+    "§9Counter-Strike III",
+    "§7Gain §a+6❈ Defense §7for §a7s",
+    "§a§7on the first hit from an",
+    "§7enemy.",
+    "",
+    "§7Use this on an item in an Anvil",
+    "§7to apply it!",
+    "",
+    "§6Source:",
+    "§aV: §7Dark Auction",
+    "",
+    "§6Applied To:",
+    "§7- §fChestplate",
+    "",
+    "§a§lUNCOMMON"
+  ],
+  "internalname": "COUNTER_STRIKE;5",
```

Comment:
can not be correct + nbt tag in both files

Reply:
thanks for catching this!